monai/data/test_time_augmentation.py

```diff
@@ -180,8 +185,10 @@ def __call__(
                 batch_output = batch_output.detach().cpu()
             if isinstance(batch_output, np.ndarray):
                 batch_output = torch.Tensor(batch_output)
-
-            transform_info = batch_data[transform_key]
+            transform_info = batch_data.get(transform_key, None)
+            if transform_info is None:
+                # no invertible transforms, adding dummy info for identity invertible
+                transform_info = [[0.0] for _ in range(self.batch_size)]
```

Comment:
I am not sure whether we should add dummy data here, maybe just leave it to `None` if no invertible transforms?

Thanks.

Reply:
`None` doesn't work with one of the collate method

Reply:
Ah, yes, that's the issue.
Or maybe we can use `"None"` to be consistent with other similar places, like:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/transforms/spatial/dictionary.py#L259

Thanks.